src/cargo/util/toml/mod.rs

```diff
@@ -1028,6 +1030,25 @@ impl TomlManifest {
             Edition::Edition2015
         };
 
+        let rust_version = project
+            .rust_version
+            .as_ref()
+            .map(|v| semver::Version::parse(&v))
```

Comment:
Would it be possible to parse partial versions?  Like `1.43`.  I think 99% of the time nobody is going to care about the patch version.  (I think it was also stipulated in the rfc.)

Reply:
This is supported now, and is tested in one of the tests.